input/fsh/research-consent-profile.fsh

```diff
@@ -63,14 +52,9 @@ Description: "The obligation of the consent."
 * extension contains 
     type 1..1 and
     reference 0..* and
-    period 0..1 and
-    parent 0..1
```

Comment:
This was added to represent a parent-child-like relationship. 
![image](https://github.com/mojitoj/ASU-research-consent-on-FHIR/assets/5555612/0703b133-3fa1-4348-b9f7-ca8815f71bd5)

Reply:
I think the parent-child relationship can be left to the codes (once there is a code for these). I think since there are only two values it's best to avoid the nesting and model them more simply. We can discuss this choice in the paper.